opentechinstitute/commotion-docs
# Move site to new template

## Summary

This is an initial migration aimed at re-integrating the new template into mainline development before the forks diverge too far. The site should function as before, but some design choices were expedient compromises and should be refined as individual issues.
## To Test
1. The new template is based on Community Technology's template. Check that the credit link in the footer is appropriate and sufficient.
2. Check for missing pages and empty content.
3. Site should render correctly in supported browsers/devices.
## Known Issues
1. Pagination is disabled. For some reason, paginator doesn't find our posts directory, so our blog index is empty when pagination is enabled.
2. Spanish and French sites are out of sync. French in particular is missing several pages.
3. Main logo height is a compromise. It's technically centered and full size, but doesn't look like it. Header should be reworked to display the logo "correctly" regardless of the amount of header text (titles, subtitles, etc.).
## Optional/Additional Tasks
1. Add Twitter link.
2. Add RSS feed. I tested the jekyll-feed plugin the other day, but output seemed buggy. Could be something in our posts or a bug in jekyll-feed.
3. Migrate to a sass-based css framework. This would give us documented, well-tested styles that could be easily customized via sass variables. The base framework could be updated automatically using bower.
4. Clean up cruft. A lot of code could be merged or thrown out.

@jheretic @georgiamoon @critzo
